### PR TITLE
Adding Base Sign Up page.

### PR DESCRIPTION
**Pull request by istutiagarwal**
Added base sign up page.

### CODE REVIEW

File: lib/app/modules/auth/login/views/auth_login_view.dart [hunk 1]
```diff
@@ -52,7 +50,7 @@ class AuthLoginView extends GetView<AuthLoginController> {
           ),
           child: FilledRoundedButton(
             text: Strings.signInWithNumber,
-            onTap: () {},
+            onTap: () {AuthSignupView.launch();},
```
Comment (A1H1) [inline lib/app/modules/auth/login/views/auth_login_view.dart:53]: make it one liner

File: lib/app/modules/auth/views/base_sign_up_view.dart [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+class BaseSignUpView extends StatelessWidget {
+  final List<Widget> children;
+
+  const BaseSignUpView({required this.children});
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Container(
+          color: AppColors.babyPink,
```
Comment (A1H1) [inline lib/app/modules/auth/views/base_sign_up_view.dart:16]: move to scaffold backgroundColor

File: lib/app/modules/auth/views/base_sign_up_view.dart [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+          margin: const EdgeInsets.only(top: Dimens.dimens_40),
+          decoration: BoxDecoration(
+              color: AppColors.white,
+              borderRadius: BorderRadius.only(
+                topRight: Radius.circular(Dimens.dimens_40),
+                topLeft: Radius.circular(Dimens.dimens_40),
+              )),
+          child: Stack(
+            children: [
+              Column(
```
Comment (A1H1) [inline lib/app/modules/auth/views/base_sign_up_view.dart:27]: use positioned.fill

**Review by A1H1 — CHANGES_REQUESTED** [review]

File: lib/app/modules/auth/views/base_sign_up_view.dart (hunk 3 above)
Comment (istutiagarwal) [reply lib/app/modules/auth/views/base_sign_up_view.dart]: done

File: lib/app/modules/auth/views/base_sign_up_view.dart (hunk 2 above)
Comment (istutiagarwal) [reply lib/app/modules/auth/views/base_sign_up_view.dart]: done

File: lib/app/modules/auth/login/views/auth_login_view.dart (hunk 1 above)
Comment (istutiagarwal) [reply lib/app/modules/auth/login/views/auth_login_view.dart]: done

**Pull request merged**